### PR TITLE
Moving Menu on TOP

### PR DESCRIPTION
**Pull request by Emily-ejag**
Adding PvMenu, deleting all Administrator Sidebars

### CODE REVIEW

**Review by richford — CHANGES_REQUESTED** [review]
@Emily-ejag, I love the change! But I'm experiencing an issue where the menu bar disappears when I refresh the page. Is this happening for you?

File: src/components/NavBar.vue [hunk 1]
```diff
@@ -18,30 +26,50 @@
...
+
+const { data: userClaims } = useQuery({
+  queryKey: ['userClaims', authStore.uid],
+  queryFn: () => fetchDocById('userClaims', authStore.uid),
+  keepPreviousData: true,
+  enabled: initialized,
+  staleTime: 5 * 60 * 1000, // 5 minutes
+});
+
+ const isSuperAdmin = computed(() => Boolean(userClaims.value?.claims?.super_admin));
```
Comment (richford) [inline src/components/NavBar.vue:52]:
```suggestion
const isSuperAdmin = computed(() => Boolean(userClaims.value?.claims?.super_admin));
```

File: src/components/NavBar.vue [hunk 2]
```diff
@@ -7,8 +7,16 @@
         </div>
       </router-link>
       <div class="login-container">
-        <!-- <i class="pi pi-bars menu-icon" @click="toggleMenu" /> -->
-        <!-- <PvMenu ref="menu" id="overlay_menu" :model="dropdownItems" :popup="true" /> -->
+        <div v-if="isSuperAdmin">
```
Comment (richford) [inline src/components/NavBar.vue:10]: Also, I think we can remove the surrounding div that restricts this to super admins. The `getSidebarActions` functions used below takes care of restricting the menu items that partner admins can see. But I think we still want them to see it.

File: src/components/NavBar.vue [hunk 3]
```diff
@@ -18,30 +26,50 @@
...
+});
+
+ const isSuperAdmin = computed(() => Boolean(userClaims.value?.claims?.super_admin));
+
+
+const isAtHome = computed(()=>{
+  return router.currentRoute.value.fullPath === '/'
+});
+const dropDownActions = computed(() => {
+  const rawActions = getSidebarActions(authStore.isUserSuperAdmin, !isAtHome.value);
```
Comment (richford) [inline src/components/NavBar.vue:59]: Let's use the computed value that you defined above.
```suggestion
  const rawActions = getSidebarActions(isSuperAdmin.value, !isAtHome.value);
```

**Review by richford — CHANGES_REQUESTED** [review]

**Comment by richford** [comment]
I think that implementing the suggestions in my second review might resolve the refresh issue that I brought up in my first review.

**Comment by Emily-ejag** [comment]
Yes, that solved it :)

**Comment by richford** [comment]
Task linked: [CU-8687342dk 1. Moving the menu button to the nav bar (using PvMenu instead of Menu to maintain consistency with the rest of the site)](https://app.clickup.com/t/8687342dk)

File: src/components/NavBar.vue (hunk 2 above)
Comment (richford) [inline src/components/NavBar.vue:10]: I still think that we need to remove this div because we want partner admins to see the menu as well.

**Review by richford — CHANGES_REQUESTED** [review]

File: src/components/NavBar.vue (hunk 2 above)
Comment (Emily-ejag) [reply src/components/NavBar.vue]: changed!

**Review by richford — APPROVED** [review]

**Pull request merged**